lib/std/genasts.nim

```diff
@@ -0,0 +1,71 @@
+import macros
```

Comment:
`macros` is under the `lib/core` directory.

Reply:
@araq to avoid changing this line back and forth, what shall i put here to unblock this PR?
I had originally written `import std/macros` but changed it to `import macros` following your recommendation in https://github.com/nim-lang/Nim/pull/17426#discussion_r598356523

Of course, if it were up to me, I'd just implement https://github.com/nim-lang/RFCs/issues/267 which would allow writing `import this/macros` which would have 0 ambiguities or scope pollution even in presence of multiple stdlib in `--path`

Reply:
The correct way is ``import macros``, inside `std` import other `std` modules via relative imports.

Reply:
well that's what i have now in this file anyways